netcore/CoreFX.issues.rsp

```diff
@@ -713,6 +713,10 @@
 # https://github.com/mono/mono/issues/15187
 -nomethod System.Diagnostics.Tests.StackFrameTests.Ctor_SkipFrames_FNeedFileInfo
 
+# JIT should not inline custom throw helpers
+-nomethod System.Diagnostics.Tests.StackTraceTests.Ctor_Exception_SkipFrames
```

Comment:
What about fixing the tests with `System.Runtime.CompilerServices.MethodImplOptions.NoInlining` ?

Reply:
@marek-safar ok, will add this change to my next set of `dotnet/corefx` changes for tests but can we merge this to keep the CI green?